src/adapters/MeetingsJSONAdapter.js

```diff
@@ -854,4 +885,59 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
 
     return concat(initialState$, settingsEvent$);
   }
+
+  /**
+   * Returns an observable that emits the display data of the video options.
+   *
+   * @param {string} ID ID of the meeting for which to update display
+   * @returns {Observable.<MeetingControlDisplay>} Observable that emits control display data
+   * @private
+   */
+  videoControl(ID) {
+    const meeting = this.datasource[ID];
+    const videoDevices = defer(() => this.getDevices('videoinput'));
+    let initialState$;
+
+    if (meeting) {
+      initialState$ = videoDevices.pipe(
+        map((devices) => ({
+          ID: SWITCH_VIDEO_DEVICE,
+          tooltip: 'Video Devices',
+          state: MeetingControlState.INACTIVE,
+          options: devices,
```

Comment:
we need to have a common format for options so that the dropdown renderer knows how to display them, eg:
- `value` for the key
- `label` for the label
- any other props are ok, eg: `device` which would hold the original object